lib/src/settings.rs

```diff
@@ -88,6 +90,7 @@ impl Default for GitSettings {
             #[cfg(feature = "git2")]
             subprocess: true,
             executable_path: PathBuf::from("git"),
+            change_id: false,
```

Comment:
I think Martin was up for just doing this by default? Maybe we don’t need a setting, unless we’re concerned about privacy implications (e.g. people not wanting to disclose that they’re using Jujutsu).

Reply:
Isn't it still useful to have a config so people can opt out if they run into problems for whatever reason? I don't mind enabling it by default, though.

Reply:
I think this should be behind a setting (at least to start!).  We don't necessarily know how this will interact with everyone's forge at this time. 

Assuming a setting is needed, I'd still lean to defaulting `false` to start with switching to to `true` in the near-mid future.

Reply:
(I think this comment belongs on `misc.toml`. The line here is only used in tests, I think.)

Reply:
> Isn't it still useful to have a config so people can opt out if they run into problems for whatever reason? I don't mind enabling it by default, though.

Right. There’s also the difference between maintaining an opt‐out forever and having one temporarily in case there are issues, so I was mostly checking the intent here.